CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.4 
+
+* Alter default host to 0.0.0.0 on development mode to improve security level [#1369](https://github.com/sinatra/sinatra/issues/1369) by David Silveira
```

Comment:
This comment does not accurately reflect the reason for this change. Technically it will make development mode less secure, as the server will accept all incoming requests.